modules/templates/helper.go

```diff
@@ -151,7 +154,6 @@ func NewFuncMap() []template.FuncMap {
 		"DiffTypeToStr":                  DiffTypeToStr,
 		"DiffLineTypeToStr":              DiffLineTypeToStr,
 		"ShortSha":                       base.ShortSha,
-		"MD5":                            base.EncodeMD5,
```

Comment:
Removing template functions in backport may cause template error if users are using customized templates.

Reply:
I didn't even consider this.

I guess this is more of a breaking change than we expected. 

We could make a note on the blog or we could backport the MD5 function back to 1.17.2 

We should also make the original PR be marked as Breaking

Reply:
I've added a breaking note to the original PR

Reply:
Should the `MD5` be added back in next 1.17? Then people will benefit from it if they skip the 1.17.2.

Reply:
Just add it back, it doesn't hurt us. We could even restore it in `main` if there is any usage of it.